chart/templates/deployment.yaml

```diff
@@ -12,34 +14,47 @@ spec:
         app: {{ .Values.appName }}
     spec:
       containers:
-        - name: {{ .Values.appName }}
-          image: "nexus-docker-public-hosted.ossim.io/unzip-and-ingest"
-          imagePullPolicy: Always
-          ports:
-            - name: http
-              containerPort: 8080
-          readinessProbe:
-            httpGet:
-              path: /health
-              port: 8080
-            initialDelaySeconds: 5
-            timeoutSeconds: 3
+        - name:  {{ .Values.appName }}
+          env:
+          {{- include "unzip-and-ingest.envVars" . | indent 8 }}
+          {{- if .Values.imageName }}
+          image: {{ .Values.imageName }}
```

Comment:
This seems unnecessary, since .Values.image.name exists

Reply:
Changed tag to be imageSkaffoldOverride to be more specific about what it is doing.